cmd/kubelet/app/options/options.go

```diff
@@ -208,6 +208,11 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.OutOfDiskTransitionFrequency.Duration, "outofdisk-transition-frequency", s.OutOfDiskTransitionFrequency.Duration, "Duration for which the kubelet has to wait before transitioning out of out-of-disk node condition status. Default: 5m0s")
 	fs.StringVar(&s.NodeIP, "node-ip", s.NodeIP, "IP address of the node. If set, kubelet will use this IP address for the node")
 	fs.BoolVar(&s.EnableCustomMetrics, "enable-custom-metrics", s.EnableCustomMetrics, "Support for gathering custom metrics.")
+	fs.StringVar(&s.CAdvisorCollectorClientCertFile, "cadvisor-collector-client-cert-file", s.CAdvisorCollectorClientCertFile, ""+
```

Comment:
I think "cAdvisor" is an implementation detail. I'd prefer to keep the flag & description generic, so it would still apply if the metric collector was moved to a different component.